src/test/dataMigration/stepsPanEuropean/countryComments/migrateCountryComments.ts

```diff
@@ -64,6 +68,9 @@ export const migrateCountryComments = async (props: Props, client: BaseProtocol)
     const tableName = `country_comments_${tableIdx}`
     Logger.info(`==== ${i + 1}/${fileNames.length} (${Numbers.format(p)})%  -> ${fileName} `)
     const table = tables[tableName]
+    if (!table) {
```

Comment:
I removed the `continue` and left the log on purpose because it was helpful to me. As the other warnings are there (row not found etc). But if you rather have it removed, happy to do so.

Reply:
sounds good! thanks